### PR TITLE
TASK: Add options to text input and area

### PR DESCRIPTION
**Pull request by samuelhauser**

### CODE REVIEW

File: packages/neos-ui-editors/src/Boolean/index.js [hunk 1]
```diff
@@ -25,18 +24,22 @@ const toBoolean = val => {
...
 
+const defaultOptions = {
+    disabled: false
+};
+
 const BooleanEditor = props => {
-    const {value, label, identifier, commit} = props;
-    const disabled = $get('options.disabled', props) || false;
+    const {value, label, identifier, commit, options} = props;
+    const finalOptions = Object.assign(defaultOptions, options);
```
Comment (skurfuerst) [inline packages/neos-ui-editors/src/Boolean/index.js]: Hey,

this line mutates the defaultOptions; so correct would be:

`const finalOptions = Object.assign({}, defaultOptions, options)`

Also needs to be fixed in the other places.

All the best,
Sebastian

File: packages/neos-ui-editors/src/TextArea/index.js [hunk 2]
```diff
@@ -2,20 +2,32 @@ import React from 'react';
...
+    disabled: false,
+    maxlength: null,
+    readonly: false
+};
+
 const TextAreaEditor = props => {
-    const {value, commit, highlight} = props;
+    const {value, commit, highlight, options} = props;
+
+    const finalOptions = Object.assign(defaultOptions, options);
```
Comment (skurfuerst) [inline packages/neos-ui-editors/src/TextArea/index.js]: same here

File: packages/neos-ui-editors/src/TextField/index.js [hunk 3]
```diff
@@ -20,19 +27,28 @@ export default class TextField extends PureComponent {
...
+    static defaultProps = {
+        options: {}
+    };
+
     render() {
         const {value, commit, validationErrors, options, i18nRegistry, highlight, onKeyPress} = this.props;
 
         // Placeholder text must be unescaped in case html entities were used
         const placeholder = options && options.placeholder && i18nRegistry.translate(unescape(options.placeholder));
+        const finalOptions = Object.assign(defaultOptions, options);
```
Comment (skurfuerst) [inline packages/neos-ui-editors/src/TextField/index.js]: same here

File: packages/neos-ui-editors/src/Boolean/index.js (hunk 1 above)
Comment (bwaidelich) [inline packages/neos-ui-editors/src/Boolean/index.js]: Woah, Spotter-of-the-day-award™ goes to Sebastian!

**Pull request merged**